network-api/networkapi/wagtailpages/pagemodels/publications/publication.py

```diff
@@ -80,3 +84,31 @@ class PublicationPage(FoundationMetadataPageMixin, Page):
         FieldPanel('contents_title'),
         FieldPanel('notes')
     ]
+
+    @property
+    def is_chapter_page(self) -> bool:
+        """Is this a chapter page (child-Publicationpage). Returns a bool."""
```

Comment:
I think we could have something more useful here as a comment, as this is already pretty obvious with the code. 

Something like: 
"the template will render differently for the page depending on if it the page is the child of another publication page (which makes it a chapter page)"

Tangental but really hoping we come up with a more intuitive way of describing these page types..